docs/api/migration-guides/local-simulators.mdx

```diff
@@ -0,0 +1,165 @@
+
+## Why are the cloud simulators being retired?
+
+The cloud simulators are being retired for several reasons:
+
+### Simulators have limitations
+
+Simulators can be useful, but they are too limited to use for research or experimentation:
+
+- Simulators are valuable for understanding small-scale quantum systems, but their usefulness maxes out at around 50 qubits, even with access to high-performance supercomputers. This ceiling comes from the exponential growth in computational resources required to simulate larger quantum systems (review [Massively parallel quantum computer simulator, eleven years later](https://arxiv.org/abs/1805.04708) for the full explanation).  Exploring quantum systems of 100 qubits and more requires hardware.
+
+- While some simulators offer noise models, it is a very hard problem to capture the entire dynamics of a real QPU. Quantum hardware offers the potential for researchers to confront the challenges inherent in quantum systems, such as noise, errors, and decoherence in a realistic testing environment.
+
+### Using quantum hardware builds unique skills
+
+Interacting with quantum hardware grows skills and experience unattainable by only using simulators: 
+
+- Direct interaction with quantum hardware builds skills because you must implement or use error mitigation or suppression techniques, for reliable computation.
+
+- Hands-on experience with quantum hardware develops a deeper understanding of quantum phenomena and how to tailor algorithms to the characteristics of quantum processors.
+
+- Engaging with quantum hardware results in practical insights into the challenges and opportunities of quantum computing, enhancing developers' ability to drive innovation in the field.
+
+## Algorithms should be adapted for quantum hardware
+
+Successful quantum algorithms must be adapted to exploit the capabilities of quantum hardware, optimizing performance and efficiency.
+
+- Quantum hardware provides a more accurate representation of real-world quantum systems than simulators.
+
+- Fine-tuning algorithms for quantum hardware involves adjusting ansatz, circuit implementations, parameters, and configuration to maximize performance.  This process is best achieved through direct experimentation with quantum hardware.
+ 
+
+## When should simulators be used?
+
+Quantum simulators should be used to help develop and test programs before fine-tuning them and sending them to quantum hardware. Local simulators can do this with good performance and efficiency.  Clifford circuits can be simulated very efficiently, and results can be verified. This is useful to give confidence in an experiment.
```

Comment:
I like this section about "when should simulators be used", could we include this detail in the verify intro page as well?